tests/ModelContextProtocol.Tests/ModelContextProtocol.Tests.csproj

```diff
@@ -1,11 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <OutputType>Exe</OutputType>
     <TargetFrameworks>net9.0;net8.0</TargetFrameworks>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <LangVersion>Latest</LangVersion>
-
+    <LangVersion>Latest</LangVersion>    
+    
```

Comment:
```suggestion
    <LangVersion>Latest</LangVersion>

```

Nit: whitespace

Reply:
> Feel free to copy the EchoTool and SampleLlmTool into the ModelContextProtocol.AspNetCore.Tests project.

I added EverythingServer as a dependency to the test project if that works.

Reply:
That should work, but I think then you'll also need to update the test since the everything echo tool is slightly different. I think we'll need to update `Assert.Contains(tools, tools => tools.Name == "Echo");` to `Assert.Contains(tools, tools => tools.Name == "echo");` and  `Assert.Equal("hello from client!", textContent.Text);` to `Assert.Equal("Echo: from client!", textContent.Text);`.